website_js_below_the_fold/readme/USAGE.rst

```diff
@@ -1,5 +0,0 @@
-This is a technical module and have no graphical interface whatsoever.
```

Comment:
Don't remove this text. Only the runbot part.

Reply:
ok, misunderstood this one. fixed now.